src/renderers/dom/fiber/wrappers/ReactDOMFiberSelect.js

```diff
@@ -101,14 +101,18 @@ function updateOptions(
     // Do not set `select.value` as exact behavior isn't consistent across all
     // browsers for all cases.
```

Comment:
I wish I knew _what_ wasn't consistent -- and where? Like was it IE8?. I'll write up an issue to do some archaeology here. 

Totally unrelated to this PR 😛.